torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1822,3 +1825,29 @@ def test_user_rrefs_confirmed_remote(self):
             args=(rref,)
         )
         self.assertEqual(ret_rref.to_here(), True)
+
+@unittest.skipIf(
+    not torch._six.PY3,
+    "Pytorch distributed autograd package " "does not support python2",
+)
+class FaultyAgentRpcTest(FaultyRpcAgentTestFixture):
```

Comment:
Let's add a test to verify that the fixture has set the backend and options properly.